Analysis/Tasks/PWGHF/qaTaskSimple.cxx

```diff
@@ -144,39 +147,104 @@ struct QAGlobalObservables {
     histograms.fill(HIST("covariance/yz"), collision.covYZ());
     histograms.fill(HIST("covariance/zz"), collision.covZZ());
 
-    int nTracks(0);
-    for (const auto& track : tracks) {
-      nTracks++;
-    }
-
     histograms.fill(HIST("multiplicity/numberOfTracks"), nTracks);
   }
 };
 
 /// Task to QA the kinematic properties of the tracks
 struct QATrackingKine {
-  o2fw::Configurable<int> nBinsPt{"nBinsPt", 100, "Number of bins for Pt"};
-  std::array<double, 2> ptRange = {0, 10.};
-  std::array<double, 2> invptRange = {-1.e2, 1.e2};
-  o2fw::Configurable<int> nBinsEta{"nBinsEta", 100, "Number of bins for the eta histogram."};
-  std::array<double, 2> etaRange = {-6, 6};
-  o2fw::Configurable<int> nBinsPhi{"nBinsPhi", 100, "Number of bins for Phi"};
+  o2fw::Configurable<int> PDGCodeSel{"PDGCodeSel", 0, "PDG code of the particle to select in absolute value, 0 selects every particle"};
+
+  o2fw::Configurable<int> PtBins{"PtBins", 100, "Number of pT bins"};
+  o2fw::Configurable<float> PtMin{"PtMin", 0.f, "Lower limit in pT"};
+  o2fw::Configurable<float> PtMax{"PtMax", 5.f, "Upper limit in pT"};
+
+  o2fw::Configurable<int> EtaBins{"EtaBins", 100, "Number of eta bins"};
+  o2fw::Configurable<float> EtaMin{"EtaMin", -3.f, "Lower limit in eta"};
+  o2fw::Configurable<float> EtaMax{"EtaMax", 3.f, "Upper limit in eta"};
+
+  o2fw::Configurable<int> PhiBins{"PhiBins", 100, "Number of phi bins"};
+  o2fw::Configurable<float> PhiMin{"PhiMin", 0.f, "Lower limit in phi"};
+  o2fw::Configurable<float> PhiMax{"PhiMax", TMath::TwoPi(), "Upper limit in phi"};
 
   o2fw::HistogramRegistry histos{"HistogramsKineQA"};
   void init(o2fw::InitContext&)
   {
-    histos.add("tracking/pt", ";#it{p}_{T} [GeV/c]", o2fw::kTH1D, {{nBinsPt, ptRange[0], ptRange[1]}});
-    histos.add("tracking/invpt", ";1/#it{p}_{T} [GeV/c]^{-1}", o2fw::kTH1D, {{nBinsPt, invptRange[0], invptRange[1]}});
-    histos.add("tracking/eta", ";#eta", o2fw::kTH1D, {{nBinsEta, etaRange[0], etaRange[1]}});
-    histos.add("tracking/phi", ";#varphi [rad]", o2fw::kTH1D, {{nBinsPhi, 0, 2 * M_PI}});
+    o2fw::AxisSpec ptAxis{PtBins.value, PtMin.value, PtMax.value};
+    o2fw::AxisSpec etaAxis{EtaBins.value, EtaMin.value, EtaMax.value};
+    o2fw::AxisSpec phiAxis{PhiBins.value, PhiMin.value, PhiMax.value};
```

Comment:
I don't understand why these `.value` are needed. If there is something that doesn't work about the configurables, it should be reported and fixed.